pages/hub/insights/index.tsx

```diff
@@ -38,8 +82,9 @@ const InsightsHub: WithPageLayout = () => {
         <div className="flex flex-col">
           <div className="flex align-middle">
             <Text>
-              We&apos;ve included a featured Insight Page for you to test out. You can also create your own to get
-              insights on repositories.
+              {session
+                ? "We&apos;ve included a featured Insight Page for you to test out. You can also create your own to get insights on repositories."
```

Comment:
Non-blocking: Is there a way to make this last abit longer? I couldn't read it within the time before it disappeared. Could address that in a future PR!

Reply:
This can be included here as it is pretty straight foward by adding a `duration` props to the toast initializer function.

Reply:
I've updated the duration to `30`secs

Reply:
I'm currently logged in, but I see the toast message about trying out the demo insights even though I already have insights. As well, the demo insights do not appear.

![CleanShot 2023-12-06 at 12 15 50](https://github.com/open-sauced/app/assets/833231/75b2dc0e-1f38-4d8e-a0d1-b1ab8548b5f9)